stable/nginx-lego/templates/default-deployment.yaml

```diff
@@ -1,21 +0,0 @@
-apiVersion: extensions/v1beta1
-kind: Deployment
-metadata:
-  name: {{ template "fullname" . }}-default-backend
-  labels:
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-spec:
```

Comment:
Doh, your right. Sorry.

What do you think of the nginx-vtx-exporter sidecar though? Still applies to the new version too.